src/Chainweb/Pact/PactService.hs

```diff
@@ -137,23 +135,6 @@ import Data.CAS (casLookupM)
 import Data.LogMessage
 import Utils.Logging.Trace
 
--- -------------------------------------------------------------------------- --
```

Comment:
This was per my review https://github.com/kadena-io/chainweb-node/pull/902#discussion_r382709468

Basically
- let's try not to add new types to PactService but in Types files
- let's try to reuse existing Exception types instead of bespoke ones